front/src/common/Loader.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC, HTMLAttributes } from 'react';
+import { useSelector } from 'react-redux';
+import cx from 'classnames';
+
+import { MainState } from 'reducers/main';
+import './Loader.scss';
+
+export const Spinner: FC<HTMLAttributes<HTMLDivElement>> = (props) => (
+  <div {...props}>
+    <div className="spinner-border" role="status">
+      <span className="sr-only">Loading...</span>
+    </div>
+  </div>
+);
+
+export const LoaderSNCF: FC<{
```

Comment:
No need to export it here as you re-export it at the end of the file. Or we can rename the file Loaders and keep it like this (and remove the export default at the end).
```suggestion
const LoaderSNCF: FC<{
```

Reply:
it was like that in the jsx file, but you've right we can remove the export at line 16

Reply:
Right, the export is not needed, (not used in the code base)